index.html

```diff
@@ -0,0 +1,99 @@
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+<!DOCTYPE html>
+<head>
+  <h1 id="title">Amazon Survey Form</form></h1>
```

Comment:
- [ ] Amazing Job so far 👏 👏 
- [ ] I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate
- [ ] I also strongly suggest that in order to sound polite you need to be using worlds like `please` `kindly` and `i suggest` and many more other worlds
- [ ] Please make amends ✌️